game.py

```diff
@@ -20,7 +20,30 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+    wrong_guesses_count = 0
```

Comment:
I like the instinct to separate out the `wrong_guesses_count` variable here as it can often be a great idea to parse out each specific variable. That being said, a list object in python actually comes with a built in variable that keeps track of this for us! As a result, we could say that `wrong_guesses_count` is actually the same thing as `len(wrong_guesses_list)`. While it isn't inherently bad to do something like this, it does mean that we are holding on to duplicate data, which is not ideal! I would recommend using the `len()` function here where you can!